test/DynamicDependency/data/Microsoft.WindowsAppRuntime.Framework/Microsoft.WindowsAppRuntime.Framework.vcxproj

```diff
@@ -77,9 +77,15 @@
     <MakeMsixInputsWithLocations Include="$(OutDir)..\RestartAgent\RestartAgent.exe">
       <TargetFile>RestartAgent.exe</TargetFile>
     </MakeMsixInputsWithLocations>
-    <MakeMsixInputsWithLocations Include="$(OutDir)..\RestartAgent\RestartAgent.exe">
+    <MakeMsixInputsWithLocations Include="$(OutDir)..\RestartAgent\RestartAgent.pdb">
```

Comment:
This change is related to DepoloymentAgent?

Reply:
No. It's a bug in RestartAgent. Noticed because DeploymentAgent started by cloning RestartAgent, problem noticed in the clone and fixed and then corrected RestartAgent too.